docs/SuccessFactors-batchsource.md

```diff
@@ -16,10 +16,13 @@ annotating metadata, etc.
 **SAP SuccessFactors Base URL (M)**: SAP SuccessFactors Base URL.
 **Entity Name (M)**: Name of the Entity to be extracted.
 
-## Credentials
+## Connection
 
 **SAP SuccessFactors Logon Username (M)**: SAP SuccessFactors Logon Username for user authentication.  
 **SAP SuccessFactors Logon Password (M)**: SAP SuccessFactors Logon password for user authentication.
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
```

Comment:
When I view the .md file, the properties appear in a single paragraph rather than on separate lines. See: 
SAP SuccessFactors Logon Username (M): SAP SuccessFactors Logon Username for user authentication.
SAP SuccessFactors Logon Password (M): SAP SuccessFactors Logon password for user authentication. Use Connection: Whether to use a connection. If a connection is used, you do not need to provide the credentials. Connection: Name of the connection to use. Entity Names information will be provided by the connection. You also can use the macro function ${conn(connection-name)}.

Also, Usually Use Connection and Connection appear in the UI before other properties. Check that the order of the properties is correct and the .md match the UI order

Reply:
Done the changes. Name and description properties are added in the connector.md file.